### PR TITLE
fix media icon diff

### PR DESCRIPTION
**Pull request by zsarnett**
## Proposed change

image
image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/dialogs/more-info/controls/more-info-media_player.ts [hunk 1]
```diff
@@ -75,14 +76,14 @@ class MoreInfoMediaPlayer extends LitElement {
               </div>
               ${supportsFeature(stateObj, SUPPORT_BROWSE_MEDIA)
                 ? html`
-                    <ha-icon-button
-                      icon="hass:folder-multiple"
+                    <mwc-icon-button
+                      class="browse-media"
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-media_player.ts:80]: Is this class used?

File: src/dialogs/more-info/controls/more-info-media_player.ts [hunk 2]
```diff
@@ -75,14 +76,14 @@ class MoreInfoMediaPlayer extends LitElement {
               </div>
               ${supportsFeature(stateObj, SUPPORT_BROWSE_MEDIA)
                 ? html`
-                    <ha-icon-button
-                      icon="hass:folder-multiple"
+                    <mwc-icon-button
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-media_player.ts:79]: import

File: src/dialogs/more-info/controls/more-info-media_player.ts [hunk 3]
```diff
@@ -75,14 +76,14 @@ class MoreInfoMediaPlayer extends LitElement {
...
+                      class="browse-media"
                       .title=${this.hass.localize(
                         "ui.card.media_player.browse_media"
                       )}
                       @click=${this._showBrowseMedia}
-                    >
-                    </ha-icon-button>
+                      ><ha-svg-icon .path=${mdiPlayBoxMultiple}></ha-svg-icon
```
Comment (bramkragten) [inline src/dialogs/more-info/controls/more-info-media_player.ts:85]: import

File: src/panels/lovelace/cards/hui-media-control-card.ts [hunk 4]
```diff
@@ -403,14 +404,16 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
                           )}
                           ${supportsFeature(stateObj, SUPPORT_BROWSE_MEDIA)
                             ? html`
-                                <ha-icon-button
+                                <mwc-icon-button
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-media-control-card.ts:407]: import

File: src/panels/lovelace/cards/hui-media-control-card.ts [hunk 5]
```diff
@@ -403,14 +404,16 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
...
                                   class="browse-media"
-                                  icon="hass:folder-multiple"
                                   .title=${this.hass.localize(
                                     "ui.card.media_player.browse_media"
                                   )}
                                   @click=${this._handleBrowseMedia}
-                                ></ha-icon-button>
+                                  ><ha-svg-icon
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-media-control-card.ts:413]: import

**Comment by zsarnett** [comment]
Relying on VScode to tell me something isn't imported really bites you sometimes

**Comment by bramkragten** [comment]
> Relying on VScode to tell me something isn't imported really bites you sometimes

Well it was imported, but through another component, it will be a problem when that element is removed...

**Review by bramkragten — APPROVED** [review]

**Pull request merged**